filter.js

```diff
@@ -139,6 +112,63 @@ Template.filter_fields.helpers({
     }
 });
 
+Template.filter_fields.onCreated(function(){
+    if ((ref = Session.get('tabular-filter')) != null ? ref.regex_value : void 0){
+        regExpValue = Session.get('tabular-filter').regex_value
+    }
+    else{
+        regExpValue = "regex"
+    }
+    if ((ref = Session.get('tabular-filter')) != null ? ref.contains_value : void 0){
+        containsValue = Session.get('tabular-filter').contains_value
+    }
+    else{
+        containsValue = "contains"
+    }
+    operators = {
+        "String":[
+            "=",
+            containsValue,
+            regExpValue
+        ],
+        "Number":[
+            "=",
+            ">",
+            "<"
+        ],
+        "Boolean": [
+            "="
+        ],
+        "Date": [
+            "=",
+            "<",
+            ">"
+        ],
+        "Object": [
+            "has value",
+            "has key"
+        ]
+    }
+    //filter_operators = ["true", "false", "AND", "OR", "FieldExpr"];
+    console.log(Session.get('tabular-filter').or_label);
```

Comment:
This console.log was creating an error, making the filter not appear. I removed it after the pull request.